### PR TITLE
tidb-functions: add info about special start_key/end_key values

### PR DESCRIPTION
**Pull request by RobertCheng-956**
### What is changed, added or deleted? (Required)
translate-docs-pr#6728

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/6728
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: functions-and-operators/tidb-functions.md [hunk 1]
```diff
@@ -108,6 +100,36 @@ select tidb_decode_key('7480000000000000FF3E5F720400000000FF0000000601633430FF33
 1 row in set (0.001 sec)
 ```
 
+表中的第一个Region以一个仅包含`table_id`的key开头。表中最后一个Region以`table_id + 1`结束。中间的Region有着更长的key，包含`_tidb_rowid`或者`handle`。
```
Comment (lilin90) [inline functions-and-operators/tidb-functions.md:103]:
```suggestion
以下示例中，表中的第一个 Region 以一个仅包含 `table_id` 的 key 开头，表中的最后一个 Region 以 `table_id + 1` 结束。中间的 Region 有着更长的 key，包含 `_tidb_rowid` 或 `handle`。
```

File: functions-and-operators/tidb-functions.md [hunk 2]
```diff
@@ -108,6 +100,36 @@ select tidb_decode_key('7480000000000000FF3E5F720400000000FF0000000601633430FF33
...
++------------+-----------------------------------------------------------+-----------------------------------------------------------+
+| stock      | {"table_id":143}                                          | {"handle":{"s_i_id":"32485","s_w_id":"3"},"table_id":143} |
+| stock      | {"handle":{"s_i_id":"32485","s_w_id":"3"},"table_id":143} | {"handle":{"s_i_id":"64964","s_w_id":"5"},"table_id":143} |
+| stock      | {"handle":{"s_i_id":"64964","s_w_id":"5"},"table_id":143} | {"handle":{"s_i_id":"97451","s_w_id":"7"},"table_id":143} |
+| stock      | {"handle":{"s_i_id":"97451","s_w_id":"7"},"table_id":143} | {"table_id":145}                                          |
++------------+-----------------------------------------------------------+-----------------------------------------------------------+
+4 rows in set (0.031 sec)
+```
+
+TIDB_DECODE_KEY 在成功解码时返回有效的JSON，在解码失败时返回传入的参数值。
```
Comment (lilin90) [inline functions-and-operators/tidb-functions.md:131]:
```suggestion
`TIDB_DECODE_KEY` 在解码成功时返回有效的 JSON，在解码失败时返回传入的参数值。
```

**Comment by lilin90** [comment]
@RobertCheng-956 Please update the content according to comments.

**Comment by RobertCheng-956** [comment]
> @RobertCheng-956 Please update the content according to comments.

OK. Ther should be a space before and after inline cold wrapped by backticks?

**Comment by lilin90** [comment]
> > @RobertCheng-956 Please update the content according to comments.
> 
> OK. There should be a space before and after inline cold wrapped by backticks?

It's one rule in the PingCAP Style Guide. See **4.1.1 空白的使用** in https://github.com/pingcap/docs-cn/blob/master/resources/pingcap-style-guide-zh.pdf.
image

**Review by lilin90 — APPROVED** [review]
/lgtm

**Comment by pushmeintheair** [comment]
> > > @RobertCheng-956 Please update the content according to comments.
> > 
> > 
> > OK. There should be a space before and after inline cold wrapped by backticks?
> 
> It's one rule in the PingCAP Style Guide. See **4.1.1 空白的使用** in https://github.com/pingcap/docs-cn/blob/master/resources/pingcap-style-guide-zh.pdf. image

Got it. Thank you.

**Pull request merged**